npm/css-src/sass/main.scss

```diff
@@ -14,3 +14,41 @@
   .hide {
   display: none;
 }
+
+.drag-and-drop {
+  margin-bottom: 40px;
+}
+
+.drag-and-drop__dropzone {
+  display: flex;
+  outline: 3px dashed #0b0c0c;
+  padding: 60px 15px;
+  text-align: center;
+  transition: outline-offset 0.1s ease-in-out, background-color 0.1s linear;
+}
+
+.drag-and-drop__dropzone [type=file] {
+  display: none;
+  left: 10.1em;
+  position: relative;
+}
+
+.drag-and-drop__dropzone p {
+  margin-bottom: 0;
+  margin-right: 10px;
+  padding-top: 7px;
+}
+
+.drag-and-drop__dropzone label {
+  display: inline-block;
+  margin-bottom: 0;
+  width: auto;
+}
+
+.file-warning-message {
+  margin-top: 20px;
+}
+
+.file-warning-text {
+  font-weight: normal;
```

Comment:
What's the reason for overriding the `<strong>` weighting?

If it's to make it look like the wireframes, then I suspect we should remove this styling and make it look like the design system instead. I don't think the wireframes are using the design system directly, so they don't always look the consistent. We should go with the default from the design system unless there's a really good reason to do something else.

Reply:
Yes, it was to look like the wireframes. Sure, I'll change it back.